x-pack/plugins/alerting/server/usage/alerts_telemetry.ts

```diff
@@ -0,0 +1,239 @@
+  );
+
+  const minConnectorsCount = Math.min(
+    ...alertsWithConnectorsCount.map((savedObject: any) => {
+      return savedObject.references.length;
+    })
+  );
+  const maxConnectorsCount = Math.max(
+    ...alertsWithConnectorsCount.map((savedObject: any) => {
+      return savedObject.references.length;
+    })
+  );
+
+  return {
+    count_total: findResult.total,
+    count_active_total: findResult.saved_objects.filter(
+      (savedObject: any) => savedObject.attributes.enabled
+    ).length,
+    count_disabled_total: findResult.saved_objects.filter(
+      (savedObject: any) => !savedObject.attributes.enabled
+    ).length,
+    throttle_time: {
+      min: alertsWithThrottle.length > 0 ? `${minThrottleTime / 1000}s` : '0',
+      avg:
+        alertsWithThrottle.length > 0
+          ? `${totalThrottleTime / alertsWithThrottle.length / 1000}s`
+          : '0',
+      max: alertsWithThrottle.length > 0 ? `${maxThrottleTime / 1000}s` : '0',
+    },
+    schedule_time: {
+      min: `${minIntervalTime / 1000}s`,
+      avg: `${(alertsWithInterval.length > 0 ? totalIntervalTime / alertsWithInterval.length : 0) /
+        1000}s`,
+      max: `${maxIntervalTime / 1000}s`,
+    },
+    connectors_per_alert: {
+      min: minConnectorsCount,
+      avg: Math.round(totalConnectorsCount / findResult.total),
+      max: maxConnectorsCount,
+    },
+  };
+}
+
+export async function getTotalCountByAlertTypes(
+  savedObjectsRepository: ISavedObjectsRepository,
+  alertTypeRegistry: AlertTypeRegistry
+) {
+  const totalByAlertType = alertTypeRegistry.list().reduce(async (accPromise: any, alertType) => {
+    const acc = await accPromise;
+    const total = await getTotalCountByAlertType(savedObjectsRepository, alertType.id);
+    return { ...acc, [alertType.name]: total };
+  }, Promise.resolve({}));
+
+  return totalByAlertType;
+}
+
+async function getTotalCountByAlertType(
+  savedObjectsRepository: ISavedObjectsRepository,
+  alertTypeId: string
+) {
+  const findResult = await savedObjectsRepository.find({
+    type: 'alert',
+    searchFields: ['alertTypeId'],
+    fields: ['alertTypeId'],
+    search: alertTypeId,
+  });
+
+  return findResult.total;
+}
+
+export async function getTotalInUseCountByAlertTypes(
+  savedObjectsRepository: ISavedObjectsRepository,
+  alertTypeRegistry: AlertTypeRegistry
+) {
+  const totalByAlertType = alertTypeRegistry.list().reduce(async (accPromise: any, alertType) => {
+    const acc = await accPromise;
+    const total = await getTotalInUseCountByAlertType(savedObjectsRepository, alertType.id);
+    return { ...acc, [alertType.name]: total };
+  }, Promise.resolve({}));
+
+  return totalByAlertType;
+}
+
+async function getTotalInUseCountByAlertType(
+  savedObjectsRepository: ISavedObjectsRepository,
+  alertTypeId: string
+) {
+  const findResult = await savedObjectsRepository.find({
+    type: 'alert',
+    searchFields: ['alertTypeId'],
+    fields: ['alertTypeId'],
+    search: alertTypeId,
```

Comment:
After doing a local test, I noticed `count_active_by_type` has `0` values for each alert. I believe the `search` attribute here doesn't work well for uuid. An alternative could be to remove `fields` and `search` and use `filter` instead with something like `alert.attributes.alertTypeId: ${alertTypeId}`.